packages/rice-uikit/package.json

```diff
@@ -12,7 +14,7 @@
   "license": "MIT",
   "scripts": {
     "start": "yarn storybook",
-    "build": "rm -rf ./dist && rollup -c && tsc -d --emitDeclarationOnly --declarationDir dist",
+    "build": "del /f dist && rollup -c && tsc -d --emitDeclarationOnly --declarationDir dist",
```

Comment:
this will break for macs please create a new script called `build:win` and leave the original
```
"build:win": "del /f dist && rollup -c && tsc -d --emitDeclarationOnly --declarationDir dist",
```